status-page/status-page-values.yaml

```diff
@@ -0,0 +1,79 @@
+kube-prometheus-stack:
+  prometheus:
+    additionalServiceMonitors:
+      - name: status-page-api
+        selector:
+          matchLabels:
+            app: status-page-api
+        namespaceSelector:
+          matchNames:
+            - status-page
+        endpoints:
+          - port: metrics
+            path: /metrics
+            interval: 10s
+      - name: status-page-database
+        selector:
+          matchLabels:
+            app: status-page-api-db
+        namespaceSelector:
+          matchNames:
+            - status-page
+        endpoints:
+          - port: metrics
+            path: /metrics
+            interval: 10s
+      - name: status-page-dex
+        selector:
+          matchLabels:
+            app: status-page-dex
+        namespaceSelector:
+          matchNames:
+            - status-page
+        endpoints:
+          - port: telemetry
+            path: /metrics
+            interval: 10s
+      - name: status-page-oathkeeper
+        selector:
+          matchLabels:
+            app: status-page-oathkeeper
+        namespaceSelector:
+          matchNames:
+            - status-page
+        endpoints:
+          - port: metrics
+            path: /metrics
+            interval: 10s
+  grafana:
+    dashboardProviders:
+      dashboardprovidersstatuspage.yaml:
+        apiVersion: 1
+        providers:
+        - name: status-page
+          folder: 'StatusPage'
+          type: file
+          disableDeletion: false
+          editable: true
+          options:
+            path: /var/lib/grafana/dashboards/status-page
+    dashboards:
+      status-page:
+        status-page-database:
+          gnetId: 13115
+          revision: 1
+          datasource:
+            - name: DS_PROMETHEUS
+              value: thanos
+        status-page-go:
+          gnetId: 10826
+          revision: 1
+          datasource:
+            - name: DS_PROMETHEUS
+              value: thanos
```

Comment:
The [go metrics dashboard](https://grafana.com/grafana/dashboards/10826-go-metrics/) is designed to be used in k8s. Is it possible, using the relabeling from `values-workload.yaml`, to add namespace and podname to the metrics, to use the selectors on the top?

This is a nice to have and is non-blocking.